src/database/adapters/mongodb.js

```diff
@@ -0,0 +1,40 @@
+const mongo = require('mongodb');
+const _ = require('lodash')
+
+module.exports = env => {
+  let _MongoClient;
```

Comment:
This is a holdout from when I was only concerned `const MongoClient = require('mongodb').MongoClient`. But now I require the entire `mongodb` library, because I need `mongodb.ObjectID`, and that just never got changed.